tox.ini

```diff
@@ -77,8 +77,23 @@ deps =
     pytest-cov
 
 [testenv:func]
+setenv =
+  TEST_JUJU_CHANNEL = 2.9/stable
 changedir = {toxinidir}/tests/functional
 commands = functest-run-suite {posargs:--keep-faulty-model}
 deps =
   git+https://github.com/openstack-charmers/zaza.git#egg=zaza
   -r {toxinidir}/tests/functional/requirements.txt
+
+[testenv:func31]
+setenv =
+  TEST_JUJU_CHANNEL = 3.1/stable
+changedir = {toxinidir}/tests/functional
+commands =
+  # NOTE(mertkirpici): functest-run-suite -b jammy focal
+  # or functest-run-suite -b jammy -b focal does not work
```

Comment:
I tried it and it's working fine for me to use `--bundle jammy focal`

Reply:
[Here](https://github.com/canonical/charm-prometheus-juju-exporter/actions/runs/6164621219/job/16730745388#step:9:251) is my trial run. If I read the output correctly, only the focal bundle was run.

`-b jammy -b focal` was a no go either.

Reply:
Can we report this as issue in zaza then? From `argparse` point of view it should work.

Reply:
yup. on it.

Reply:
Thanks :)